DataProcessing/process.py

```diff
@@ -0,0 +1,96 @@
+# 1. Download the data
+# 2. process (if required)
+# 3. Output data by date tp /temp-output-directory/alternative/quiver/twitter/universe/{date}.csv
+# 4. Output data for each ticker to /temp-output-directory/alternative/quiver/twitter/{symbol}.csv
+
+import os
+from clr_loader import get_coreclr
+from pythonnet import set_runtime
+set_runtime(get_coreclr(os.path.join(os.path.dirname(os.path.realpath(__file__)), "config.json")))
+
+from AlgorithmImports import *
+from QuantConnect.Lean.Engine.DataFeeds import *
+AddReference("Fasterflect")
+
+import pathlib
+import requests
+import shutil
+import time
+
+VendorName = "quiver";
+VendorDataName = "twitter";
+
+class QuiverQuantTwitterFollowersDataDownloader:
+    def __init__(self, destinationFolder, canCreateUniverseFiles, apiKey = None):
+        self.destinationFolder = os.path.join(destinationFolder, VendorDataName)
+        self.universeFolder = os.path.join(self.destinationFolder, "universe")
+        self.canCreateUniverseFiles = canCreateUniverseFiles
+        self.clientKey = apiKey if apiKey else os.environ["QUIVER_API_KEY"]
+
+        if os.path.exists(self.universeFolder):
+            shutil.rmtree(self.universeFolder)
+    
+        pathlib.Path(self.destinationFolder).mkdir(parents=True, exist_ok=True)
+        pathlib.Path(self.universeFolder).mkdir(parents=True, exist_ok=True)
+
+    def Run(self):
+
+        mapFileProvider = LocalZipMapFileProvider()
+        mapFileProvider.Initialize(DefaultDataProvider());
+
+        companies = sorted(self.HttpRequester("companies"), key=lambda x: x['Ticker'])
+    
+        for c in companies:
+            ticker = c['Ticker']
+            filename = os.path.join(self.destinationFolder, f'{ticker.lower()}.csv')
+            print(f'Processing ticker: {ticker}')
+            i = 5
+
+            while i != 0:
```

Comment:
Think we should expand variable names `i`,`c`, `dt`, `fp` etc to be more representative when possible